### PR TITLE
Add 2020 Methodology in Japanese

### PR DESCRIPTION
**Pull request by bazzadp**
This is liable to change some more but at the moment it's very similar to 2019 and we've committed in English to think we can do the same in the other languages if one of the Japanese speakers is willing to review.

Will point out paragraphs that need particular review

### CODE REVIEW

File: src/templates/ja/2020/methodology.html [hunk 1]
```diff
@@ -0,0 +1,426 @@
...
+          HTTP Archiveは、<a href="#chrome-ux-report">Chrome UXレポート</a> からウェブサイトのURLを取得しています。Chrome UXレポートはGoogleが公開しているデータセットで、Chromeユーザーが積極的に訪問している数百万のウェブサイトのユーザー体験を集計しています。これにより、実際のウェブ利用状況を反映した最新のウェブサイトのリストが得られます。Chrome UXレポートデータセットにはフォーム ファクター ディメンションが含まれており、デスクトップ ユーザーやモバイル ユーザーがアクセスしたすべてのWebサイトを取得するために使用します。
+        </p>
+
+        <p>
+          Web Almanacが使用した2020年8月のHTTP Archiveのクロールでは、Webサイトのリストに、最新のChrome UXレポートリリースである2020年6月（202006）を使用しました。このデータセットは2020年7月14日にリリースされたもので、6月中にChromeユーザーが訪問したウェブサイトをキャプチャしています。
+        </p>
+
+        <!-- To be translated <p>
+          There was was a 20-30% growth on websites analysed compared to the <a href="../2019/methodology#websites">number of sites analysed for the 2019 Web Almanac</a>. This increase has been analysed by <a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a> in his <a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">Growth of the Web in 2020</a> post.
+        </p> -->
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:128]: Can a native speaker translate this whole section and remove the `<!--` and `-->` marks?

File: src/templates/ja/2020/methodology.html [hunk 2]
```diff
@@ -0,0 +1,426 @@
...
+      </p>
+
+      <p>
+        Throughout August 2020, the HTTP Archive data pipeline crawled several million websites, gathering the metadata to be used in the Web Almanac.
+      </p>
+
+      <p>
+        The data analysts began writing queries to extract the results for each metric. In total, <strong>TBC queries</strong> were written by hand! You can browse all of the queries by chapter in the <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> directory of the project's GitHub repository.
+      </p>
+    </section>-->
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:397]: Can a native speaker translate this whole section and remove the `<!--` and `-->` marks?

File: src/templates/ja/2020/methodology.html [hunk 3]
```diff
@@ -0,0 +1,426 @@
...
+      <p>
+        最後に、編集者は各章を修正し、簡単な文法的な誤りを修正し、読書体験全体に一貫性を持たせるようにしました。
+      </p>
+    </section>
+
+    <section>
+      <h2 id="looking-ahead"><a href="#looking-ahead" class="anchor-link">将来を見据えて</a></h2>
+
+      <p>
+        Webアルマナックの2020年版は、内省と前向きな変化への取り組みというWebコミュニティの毎年恒例の伝統の2番目です。ここまで来れたのは、多くの献身的な<a href="{{ url_for('contributors', year=year, lang=lang) }}">コントリビューター</a>のおかげで、記念碑的な努力をしてきました。
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:419]: This line could do with a particular review

File: src/templates/ja/2020/methodology.html (hunk 1 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:128]: Yes, I think can.

File: src/templates/ja/2020/methodology.html (hunk 2 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:397]: Yes, I think can.

File: src/templates/ja/2020/methodology.html (hunk 3 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:419]:
```suggestion
        Web Almanacの2020年版は毎年恒例、Webコミュニティ伝統の2回目で、内省と前向きな変化への取り組みです。ここまで来れたのは、多くの献身的な<a href="{{ url_for('contributors', year=year, lang=lang) }}">コントリビューター</a>のおかげで、記念碑的な努力をしてきました。
```

**Review by ksakae1216 — COMMENTED** [review]
I have a comment.

**Review by ksakae1216 — APPROVED** [review]

**Comment by bazzadp** [comment]
I was still waiting for suggested text for the other two pieces @ksakae1216 ?

File: src/templates/ja/2020/methodology.html (hunk 1 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:128]:
```suggestion
<p>2019年のWeb Almanac</a>のために分析した<a href="../2019/methodology#websites">サイト数と比較して、分析したウェブサイトで20～30%の増加がありました。この増加は、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='paulcalvano') }}">Paul Calvano</a>が<a href="https://paulcalvano.com/2020-09-29-growth-of-the-web-in-2020/">Growth of the Web in 2020</a>の投稿で分析したものです。</p>
```

File: src/templates/ja/2020/methodology.html (hunk 2 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:397]:
```suggestion
    <section>
      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>

      <p>
        2020年Web Almanacは、2020年6月に<a href="https://twitter.com/rick_viscomi/status/1273135952848977920?s=20">新規執筆者を募集して</a>スタートしました。ブレーンストーミングのラウンドの後、22の章が固まり、私たちは<a href="https://github.com/HTTPArchive/almanac.httparchive.org/issues/876">各章に主題専門家とレビュワーを割り当てる作業を開始しました</a>。このプロセスには、この規模のプロジェクトにコミットするためにボランティアを集めるという課題があったため、いくつかの固有の偏りがありました。したがって、寄稿者の多くは同じ専門家サークルのメンバーです。Web Almanacの今後のエディションの明確な目標の1つは、著者やレビュワーとして、代表性の低い、異質な声をより多く取り入れることを奨励することです。
      </p>

      <p>
        私たちは2020年7月に、人々と支部のペアリングを行い、各支部を構成する個々の指標を最終的に決定するために、彼らの意見を聞きました。
      </p>
    </section>

    <section>
      <h3 id="analysis"><a href="#analysis" class="anchor-link">Analysis</a></h3>

      <p>
        2020年7月と8月に、メトリクスとチャプターの安定したリストを持って、データアナリストは、実現可能性のためにメトリクスをトリアージしました。場合によっては、分析能力のギャップを埋めるために、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>を作成する必要がありました。
      </p>

      <p>
        2020年8月を通して、HTTPアーカイブデータパイプラインは数百万のウェブサイトをクロールし、Web Almanacで使用するためのメタデータを収集しました。
      </p>

      <p>
        データアナリストは、各メトリクスの結果を抽出するためのクエリを書き始めました。合計すると、<strong>TBC クエリ</strong>が手書きで書かれていました! プロジェクトのGitHubリポジトリの <a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020">sql/2020</a> ディレクトリで、章ごとにすべてのクエリを閲覧できます。
      </p>
    </section>
```

**Review by ksakae1216 — COMMENTED** [review]
Sorry, I translated  two pieces.

**Pull request merged**